server/models/wave_datum.py

```diff
@@ -0,0 +1,28 @@
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
```

Comment:
If `poetry run check` fails on this line, add `type: ignore`.

sqlalchemy will provide type support in version 2.0.